packages/venia-ui/lib/components/SignIn/__tests__/signIn.spec.js

```diff
@@ -114,7 +114,8 @@ test('displays an error message if there is a sign in error', () => {
     expect(component.toJSON()).toMatchSnapshot();
 });
 
-test('calls `signIn` on submit', () => {
+// TODO: Move this to useSignIn.spec.js and test handleSignIn
+test.skip('calls `signIn` on submit', () => {
```

Comment:
Are you tackling this in this PR? 
If not, mind creating a ticket for it?

Reply:
Do we make tickets for writing tests?

Reply:
https://jira.corp.magento.com/browse/PWA-700

Reply:
> Do we make tickets for writing tests?

Lol fair enough. Thanks :)